<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CclosingBalanceDlg</class>
 <widget class="QDialog" name="CclosingBalanceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_saveCloseBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_closingDate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_closingasonlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Closing As on</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>661</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Departtment/Account</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Closing Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cumulative Total</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
